src/main.rs

```diff
@@ -337,58 +382,55 @@ fn process_file(
         first_line = false;
 
         // that's a bad workaround
-        if line.trim() == "#[rustfmt::skip]" {
+        // Why do we require this, we dont use it (?)
+        if trimmed == "#[rustfmt::skip]" {
```

Comment:
at some point the template contained that because reformatting the code added line-breaks which made the replace-string fail - I guess having it for the future is a good thing